package.json

```diff
@@ -68,4 +68,4 @@
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.3.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
But here. CCA removes the end of line automatically, however, `prettier` thinks adding a newline to the end of file is recommended (https://github.com/prettier/prettier/issues/2575) and this behaviour is not configurable.